src/com/moclojer/specs/moclojer.clj

```diff
@@ -28,61 +40,98 @@
   (let [{:keys [content]} (render-template (:path external-body) request)]
     (assoc external-body :path content)))
 
+(defn parse-json-safely
+  "Tenta fazer parse de um JSON, retornando o conteúdo original em caso de erro"
+  [content]
+  (try
+    (json/read-str content)
+    (catch Exception e
+      (log/log :warn :json-parse-error
+               :body content
+               :message (.getMessage e))
```

Comment:
Is a log necessary? It's an user error, so it's not really an exception.

Reply:
that's why it's `log` and not `error`